### ISSUE TITLE
Home Assistant Energy Page

### ISSUE THREAD

**Issue by iannecj** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#80]
This had been reporting on Energy Page completely however now while bars in individual devices are still reporting, the total grid energy graph is not.  When I go to Energy configuration in home assistant I see its complaining:

Last reset missing
The following entities have state class 'measurement' but 'last_reset' is missing:
sensor.pm_main_total_daily_kwh_farmbell

Is there new configuration in ESPhome YAML file?  I don't see this last reset aspect in the docs.

Here is my current YAML for these sensors:

`#kWh Farmbell Daily
  - platform: total_daily_energy
    name: ${disp_name} Total Daily kWh Farmbell
    power_id: totalWatts
    filters:
      - multiply: 0.001
    unit_of_measurement: kWh
    device_class: "energy"
    state_class: "measurement"

#kWh Pool
  - platform: total_daily_energy
    name: ${disp_name} Total Daily kWh Pool
    power_id: totalWattsPool
    filters:
      - multiply: 0.001
    unit_of_measurement: kWh

#kWh Upstairs AC
  - platform: total_daily_energy
    name: ${disp_name} Total Daily kWh Upstairs AC
    power_id: totalWattsUpAC
    filters:
      - multiply: 0.001
    unit_of_measurement: kWh
    
#kWh Downstairs AC
  - platform: total_daily_energy
    name: ${disp_name} Total Daily kWh Downstairs AC
    power_id: totalWattsDownAC
    filters:
      - multiply: 0.001
    unit_of_measurement: kWh
    
switch:
  - platform: restart
    name: ${disp_name} Restart  
time:
  - platform: sntp
    id: sntp_time `

Meter has been flawless by the way thanks!

Thanks,

Chris

**Comment by CircuitSetup** [comment]
[It looks like there was a fix to the default behavior of Total Daily Energy.](https://github.com/esphome/esphome/pull/2163) 

Try changing _state_class: measurement_ to _state_class: total_increasing_

**Comment by CircuitSetup** [comment]
Yes, I'll definitely change it in the examples. Thanks for bringing this to light!

**Comment by CircuitSetup** [comment]
oh, I forgot to ask. Was state_class added automatically (as stated in the forum thread), or is that something you added? I'm trying to figure out if it explicitly needs to be in the config.

**Comment by iannecj** [comment]
I had for the main total energy sensor in YAML and I had added it at some point.  I believe from an online example.
device_class: "energy"
state_class: "measurement"

That did not work and it complained of no last reset parameter as we said.

Based on your suggestion I changed to:
    device_class: "energy"
    state_class: "total_increasing"

And it works but it may have also worked if I had just deleted  state_class: "measurement".  Didn't try that.
